### PR TITLE
add input DTO validation

### PR DESCRIPTION
**Pull request by shinehlina**

### CODE REVIEW

File: taskcoordinator/app/src/main/java/com/microservices/taskcoordinator/dto/inbound/OrderSubmittedDto.java [hunk 1]
```diff
@@ -3,15 +3,23 @@
...
 @Setter
 @AllArgsConstructor
 @EqualsAndHashCode
+@NoArgsConstructor
 public class OrderSubmittedDto {
 
+    @NotNull
     private Integer orderId;
 
+    @Valid
```
Comment (AfanasievYuriy) [inline taskcoordinator/app/src/main/java/com/microservices/taskcoordinator/dto/inbound/OrderSubmittedDto.java:22]: ?

Comment (shinehlina) [reply taskcoordinator/app/src/main/java/com/microservices/taskcoordinator/dto/inbound/OrderSubmittedDto.java:22]: Annotation in InboundLaundryStateDto would not be checked without that annotation

Comment (AfanasievYuriy) [reply taskcoordinator/app/src/main/java/com/microservices/taskcoordinator/dto/inbound/OrderSubmittedDto.java:22]: ohh
i see. sorry

**Review by AfanasievYuriy — APPROVED** [review]
Good job! Thanks!

**Pull request merged**